nikola/nikola.py

```diff
@@ -1660,6 +1660,11 @@ def generic_rss_renderer(self, lang, title, link, description, timeline, output_
             language=lang
         )
 
+        copyright = self.config['CONTENT_FOOTER'](lang)
```

Comment:
The existence of `copyright` should be configurable, and `CONTENT_FOOTER` might not be the right place to take it from.